### PR TITLE
Implemented Product model and rspec for the same

### PR DESCRIPTION
**Pull request by rineshabana**
Added javascript alert for for clicking on User Address without creating User Profile.
Implemented Product model ,added data in seed file and wrote rspec for product model.

### CODE REVIEW

File: app/controllers/user_address_controller.rb [hunk 1]
```diff
@@ -1,8 +1,13 @@
 class UserAddressController < ApplicationController
+  protect_from_forgery except: :index
```
Comment (stingrayzboy) [inline app/controllers/user_address_controller.rb:2]: Why did you add this protect_from_forgery ?

Comment (stingrayzboy) [inline app/controllers/user_address_controller.rb:2]: Why did you add this protect from forgery?

File: app/models/product.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class Product < ApplicationRecord
+  validates :name, presence: true
+  validates :price, presence: true
```
Comment (stingrayzboy) [inline app/models/product.rb]: Also add a validation of price being a number and always positive.

File: db/seeds.rb [hunk 3]
```diff
@@ -18,3 +18,7 @@
                                    city: 'City', state: 'State', pincode: '400241', country_code: '+91', contact_number: '1234567890')
 
 puts 'Address Created'
+
+product = Product.create(name:'Apple', price: 200)
+
+puts 'Product Created'
```
Comment (stingrayzboy) [inline db/seeds.rb]: Always put a empty line at the last.

**Review by stingrayzboy — CHANGES_REQUESTED** [review]
Please address the changes.

File: app/controllers/user_address_controller.rb (hunk 1 above)
Comment (rineshabana) [reply app/controllers/user_address_controller.rb:2]: To prevent `CSRF` attack.

File: app/models/product.rb (hunk 2 above)
Comment (rineshabana) [reply app/models/product.rb]: done.

File: db/seeds.rb (hunk 3 above)
Comment (rineshabana) [reply db/seeds.rb]: Done.

**Review by stingrayzboy — APPROVED** [review]
Looks Good.
